proto/cosmos/bank/v1beta1/bank.proto

```diff
@@ -10,8 +10,12 @@ option go_package = "github.com/cosmos/cosmos-sdk/x/bank/types";
 
 // Params defines the parameters for the bank module.
 message Params {
-  option (gogoproto.goproto_stringer)       = false;
-  repeated SendEnabled send_enabled         = 1;
+  option (gogoproto.goproto_stringer) = false;
```

Comment:
It's almost correct. In the comment we need to add a version pointing from when it is deprecated.

Reply:
Updated the comment to
```proto
  // Deprecated: Use of SendEnabled in params is deprecated.
  // For genesis, use the newly added send_enabled field in the genesis object.
  // Storage, lookup, and manipulation of this information is now in the keeper.
  // As of v0.47.0, it only exists here for backwards compatibility of genesis files.
```

Reply:
For what it's worth, none of the other deprecated proto messages or fields have similar version statements.

It's tough to get it correct right now. There's just no way of knowing now, which version will contain this change.

Here's some option analysis:
1. Use "As of v0.47.0 ...".
   **PRO**: If it's released in v0.47.0, everything is good and we've provided a nice shortcut to this info.
   **CON**: If it's released later, it can cause confusion. E.g. A person sees the comment (in maybe v0.48.0), grabs v0.47.0 and it's not actually deprecated there.
2. Use "Only exists after v0.46 ...".
   **PRO**: No matter which version the change is released in, this would _technically_ be a correct statement.
   **CON**: If not released in v0.47.0, it can still cause the same confusion mentioned before. A person might see the statement in v0.48.0, then notice it missing from v0.47.0.
3. Don't set it now. Instead, create an issue to have the comment added closer to when it's released. That issue could also ask that similar comments be added to the other deprecated proto messages and fields.
   **PRO**: There's much less chance that it's incorrect.
   **CON**: Someone wondering which version it was deprecated in will have to take an extra step and dig through the change log.
4. Omit the version comment entirely.
   **PRO**: don't have to worry about getting it right.
   **CON**: anyone wanting to know the last usable version (or first unusable version) would have to search the change log.

I'm personally partial to picking point 3. My second choice would be 4. I'd rather not have the information there at all than to have it be incorrect. The info will be available in the change log either way, but contradictory info can cause problems.

One factor to weigh when comparing 1/2 vs 3/4 is the odds that this isn't released in v0.47.0. I feel like there's about a 10% chance of that. That's kind of based on the amount of stuff that's changing in v0.46.0, but mostly just gut feeling (i.e. completely made up). You all probably have more accurate knowledge and feelings on that.

Reply:
I would go with `Only exists after v0.46 ...`, because I say 0.47 won't exist because we're actually aiming for 1.0

Reply:
Updated it to
```proto
// Deprecated: Use of SendEnabled in params is deprecated.
// For genesis, use the newly added send_enabled field in the genesis object.
// Storage, lookup, and manipulation of this information is now in the keeper.
// This version only still exists after v0.46 for backwards compatibility of
// genesis files.
```

Reply:
We don't know what the next release will be, so we can't use 0.47, so that's why "next release after 0.46" makes the most sense.